app/views/ctc/questions/income/edit.html.erb

```diff
@@ -1,7 +1,8 @@
-<% content_for :form_question, t("views.ctc.questions.income.title", tax_year: Rails.application.config.current_tax_year) %>
+<% content_for :form_question, t("views.ctc.questions.income.title", current_tax_year: current_tax_year) %>
 <% content_for :form_help_text do %>
-  <p class="form-question--tabbed"><%=t("views.ctc.questions.income.filing_single") %></p>
-  <p class="form-question--tabbed"><%=t("views.ctc.questions.income.filing_married") %></p>
+  <% standard_deduction = StandardDeduction.new(tax_year: current_tax_year) %>
+  <p class="form-question--tabbed"><%=t("views.ctc.questions.income.filing_single", single_deduction: number_to_currency(standard_deduction.standard_deduction(:single))) %></p>
```

Comment:
Great work removing the hard-coded number from the translation string, here and in a few other places.